### ISSUE TITLE
Enable bulk editing

### ISSUE THREAD

**Issue by richhiggins** [contentful/contentful-migration#99]
It would be useful if bulk editing flag could be set as part of the content model definition. 

At present I can't see a way to do this.

thanks!

**Comment by Khaledgarbaya** [comment]
Hi @richhiggins,
Can you please provide more details. Do you mean something like this https://github.com/contentful/contentful-migration/blob/master/examples/03-long-example.js ?

Best,
Khaled

**Comment by richhiggins** [comment]
Hey, no I don't think its that. I mean enable this to be set via the CLI

![image](https://user-images.githubusercontent.com/3781432/39920513-303fc16c-550f-11e8-9647-9d7ecef6014c.png)

**Comment by zcei** [comment]
This is defined in an "Editor Interface" for that content type, you're looking to set the `bulkEditing` key there to true:

```json
    {
      "fieldId": "lessons",
      "settings": {
        "helpText": "The lessons this course consists of.",
        "bulkEditing": true
      },
      "widgetId": "entryLinksEditor"
    }
```

**Comment by connor-baer** [comment]
@zcei Doesn't this PR need to be merged first? https://github.com/contentful/contentful-migration/pull/97

Or did I miss that "Editor Interfaces" are already supported?

**Comment by zcei** [comment]
Ah damn, it's a setting..

(There has been some work on editor interfaces (#58) and a fix in #89, that's why I was thinking we're there yet.)

With #97 my answer will become valid 😛

**Comment by Khaledgarbaya** [comment]
Hey @richhiggins it is possible and we are working on supporting that in https://github.com/contentful/contentful-migration/pull/97

**Comment by dlitvakb** [comment]
It is now added!

**Comment by Khaledgarbaya** [comment]
this is now available in `contentful-migration@0.8.0`.

**Comment by richhiggins** [comment]
awesome, now using it!
